resources/helm/v2.1/istio-init/files/servicemeshextensions.maistra.io.crd.yaml

```diff
@@ -1,50 +1,53 @@
-
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   labels:
     maistra-version: "2.1.0"
-    chart: istio
-    release: istio
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.0
+    controller-gen.kubebuilder.io/version: v0.2.4
   creationTimestamp: null
   name: servicemeshextensions.maistra.io
 spec:
   group: maistra.io
   names:
-    categories:
-    - maistra-io
     kind: ServiceMeshExtension
     listKind: ServiceMeshExtensionList
     plural: servicemeshextensions
-    shortNames:
-    - sme
```

Comment:
losing shortname and category :slightly_frowning_face: we definitely have to move this to maistra/api and use code-gen 0.4.0+